app/assets/javascripts/3dbio_viewer/src/webapp/components/network/NetworkForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useState, useRef } from "react";
+import i18n from "../../utils/i18n";
+import { Dropzone, DropzoneRef } from "../dropzone/Dropzone";
+import IncludeNeighborsCheckbox from "./IncludeNeighborsCheckbox";
+import Label from "./Label";
+import NetworkExample from "./NetworkExample";
+import UniProtAccessionTextArea from "./UniProtAccessionTextArea";
+import SpeciesSelect from "./SpeciesSelect";
+import "./Network.css";
+
+interface NetworkForm {
+    species: string;
+    uniProtAccession: string;
+    includeNeighboursWithStructuralData: boolean;
+}
+
+function getInitialNetworkForm(): NetworkForm {
+    return {
+        species: "homoSapiens",
+        uniProtAccession: "",
+        includeNeighboursWithStructuralData: false,
+    };
+}
```

Comment:
When no arguments are required to build the initial value, we can simply: `const initialNetworkForm: NetworkFrom = {...}`